x-pack/plugins/security_solution/public/overview/components/stat_value.tsx

```diff
@@ -40,7 +40,7 @@ const StatValueComponent: React.FC<{
   }, [isLoading, isInitialLoading, setIsInitialLoading]);
 
   return (
-    <EuiFlexGroup alignItems="center" gutterSize="none">
+    <EuiFlexGroup alignItems="center" gutterSize="none" responsive={false}>
```

Comment:
Puts the number and bar on same line on small screens.
Before:
<img width="491" alt="Screen Shot 2022-05-10 at 10 33 45 AM" src="https://user-images.githubusercontent.com/6935300/167678462-8443c04b-8d1d-4a8b-aaf7-e6b153f2e69e.png">
After:
<img width="434" alt="Screen Shot 2022-05-10 at 10 33 50 AM" src="https://user-images.githubusercontent.com/6935300/167678478-5b9ad821-ba95-47d0-90ed-5aefc439d54d.png">